docs/design/terms_and_abbreviations.tex

```diff
@@ -2,4 +2,50 @@
 
 \section*{Terms and Abbreviations}
 
+\begin{tabular}[h]{rp{0.75\linewidth}}
+    \hline
+    \textbf{Term} & \textbf{Definition}\\
+    \hline
+
+    ANN & Artificial Neural Network, or simply ``Neural Network'', is a data processing model modeled after neuron interactions. The process consists of forward propagation through the use of several matrix multiplications.\cite{ann}\\
```

Comment:
Remove

Reply:
why do you want to remove ANN?

Reply:
I don't think it's used in the document, is it?

Reply:
I didn't think that that mattered, but I'll remove it.